lblod/ember-rdfa-editor-lblod-plugins
# GN-4418: display number numerically if too big to convert to words

### Overview
When trying to convert a number which is too big using the `n2words` function, `n2words` throws an error. This caused the editor to crash.
This PR introduces a wrapper around `n2words` which displays a warning if `n2words` throws an error and returns the provided number numerically.
This ensures that numbers which are too big to be represented in words are always displayed numerically.
Additionally, the 'to words' toggle is disabled if it is not possible to convert the current number to words.

### Linked issues
- Jira ticket: https://binnenland.atlassian.net/browse/GN-4418?atlOrigin=eyJpIjoiNmU4MzlkZDVkMzgwNDBlYmI0YTI2NmVkYjljYWIzZTkiLCJwIjoiaiJ9 

### How to test/reproduce
Insert a number and input a high value (above 999 trillion) and select the 'to words' toggle. The number should still be displayed numerically and an error should no longer be thrown.

### Checks PR readiness
